src/calc-length.js

```diff
@@ -93,6 +93,70 @@
     return new CalcLength(calcDictionary);
   };
 
+  CalcLength.prototype._addCalcLengths = function(addedLength) {
+    if (!(addedLength instanceof CalcLength)) {
+      throw new TypeError('Objects not of type CalcLength');
```

Comment:
Maybe make the message 'Argument must be a CalcLength'. I assume this was copied from SimpleLength; feel free to also change it there.